discovery-provider/plugins/network-monitoring/src/content/queries.ts

```diff
@@ -311,12 +317,12 @@ export const getSecondary1WalletBatch = async (
     limit: number,
 ): Promise<string[]> => {
     const walletBatchResp: unknown[] = await sequelizeConn.query(`
-        SELECT wallet 
-        FROM network_monitoring_users
-        WHERE run_id = :run_id
-        AND secondary1spid = :spid
-        OFFSET :offset
-        LIMIT :limit;
+    SELECT wallet 
+    FROM network_monitoring_users
+    WHERE run_id = :run_id
+    AND secondary1spid = :spid
```

Comment:
aren't these 3 functions just the same query if you template `columnName` with `primaryspid` `secondary1spid`, etc

same for the saveBatch fns below

Reply:
They are and actually in an earlier iteration of the code I had it like that. I can't remember what prompted me to change it to be like this but I can change it back if you think that would be cleaner